package.json

```diff
@@ -20,15 +20,15 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
     "chai-datetime": "^1.4.1",
-    "mocha": "^3.2.0"
+    "mocha": "^6.1.3"
   },
   "dependencies": {
     "lodash": "^4.17.1",
     "pg": "^6.1.3",
     "semver": "^5.3.0",
-    "sequelize": "3.30.2 - 4"
+    "sequelize": "3.30.2 - 5"
```

Comment:
I *think* if we support 5 now this can just be `>=3.30.2`? My syntax might be wrong.

Reply:
Hm, I'm not sure. Do we want to blindly commit to supporting version 6 when it eventually comes out?

Reply:
Yeah, I guess not, I just remember the use of `-` being something of a hack, but I guess it's better this way.

Reply:
👍